front/app/components/admin/PostManager/InputManager.tsx

```diff
@@ -200,7 +200,7 @@ const InputManager = ({
     setQueryParameters({ ...queryParameters, 'page[number]': 1, assignee });
   };
 
-  const onChangeFeedbackFilter = (feedbackNeeded: boolean) => {
+  const onChangeFeedbackFilter = (feedbackNeeded: boolean | undefined) => {
```

Comment:
Not sure how this didn't show an error anywhere because we pass undefined in FeedbackToggle 🤔

Reply:
This same problem happens in on onChange handlers... Types are not correctly defined so would be easy to introduce bugs. May push up another PR later to fix this. This whole InputManager implementation 🥲

Reply:
That would be great 😊